include/solvers/vof.h

```diff
@@ -392,6 +404,9 @@ class VOF : public AuxiliaryPhysics<dim, TrilinosWrappers::MPI::Vector>
     parallel::distributed::SolutionTransfer<dim, TrilinosWrappers::MPI::Vector>>
     previous_solutions_transfer;
 
+  // Conservation Analysis
+  ConvergenceTable error_table_fs;
```

Comment:
You can use a regular TableHandler here, because you are not monitoring an error or something that converges.

Reply:
done